pkg/apis/contrail/v1alpha1/tests/config_customized_test.go

```diff
@@ -13,38 +15,35 @@ import (
 
-	t.Run("analytics ttl settings in the configmap data are the same as in the config spec", func(t *testing.T) {
-		environment := SetupEnv()
-		cl := *environment.client
-		environment.configResource.Spec.ServiceConfiguration.AnalyticsConfigAuditTTL = makeIntPointer(111)
-		environment.configResource.Spec.ServiceConfiguration.AnalyticsDataTTL = makeIntPointer(222)
-		environment.configResource.Spec.ServiceConfiguration.AnalyticsFlowTTL = makeIntPointer(333)
-		environment.configResource.Spec.ServiceConfiguration.AnalyticsStatisticsTTL = makeIntPointer(444)
-
-		err := environment.configResource.InstanceConfiguration(reconcile.Request{types.NamespacedName{Name: "config1", Namespace: "default"}}, &environment.configPodList, cl)
-		if err != nil {
-			t.Errorf("get configmap: (%v)", err)
-		}
-		err = cl.Get(context.TODO(),
-			types.NamespacedName{Name: "config1-config-configmap", Namespace: "default"},
-			&environment.configConfigMap)
-		if err != nil {
-			t.Errorf("get configmap: (%v)", err)
-		}
+	require.NoError(t, environment.configResource.InstanceConfiguration(reconcile.Request{
+		types.NamespacedName{Name: "config1", Namespace: "default"}}, &environment.configPodList, cl), "Error while configuring instance")
 
-		if environment.configConfigMap.Data["collector.1.1.1.1"] != customizedCollectorConfig {
-			diff := diff.Diff(environment.configConfigMap.Data["collector.1.1.1.1"], customizedCollectorConfig)
-			t.Errorf("get collector config: \n%v\n", diff)
-		}
+	require.NoError(t, cl.Get(context.TODO(), types.NamespacedName{Name: "config1-config-configmap", Namespace: "default"},
+		&environment.configConfigMap))
 
-		if environment.configConfigMap.Data["queryengine.1.1.1.1"] != customizedQueryEngineConfig {
-			diff := diff.Diff(environment.configConfigMap.Data["queryengine.1.1.1.1"], customizedQueryEngineConfig)
-			t.Errorf("get queryengine config: \n%v\n", diff)
-		}
+	t.Run("custom queryengine config settings", func(t *testing.T) {
+		queryEngine, err := ini.Load([]byte(environment.configConfigMap.Data["queryengine.1.1.1.1"]))
+		require.NoError(t, err, "Error while reading config")
+		assert.Equal(t, "222", queryEngine.Section("DEFAULT").Key("analytics_data_ttl").String(), "Invalid analytics_data_ttl")
+	})
 
+	t.Run("custom collector config settings", func(t *testing.T) {
+		collector, err := ini.Load([]byte(environment.configConfigMap.Data["collector.1.1.1.1"]))
+		require.NoError(t, err, "Error while reading config")
+		assert.Equal(t, "222", collector.Section("DEFAULT").Key("analytics_data_ttl").String(), "Invalid analytics_data_ttl")
+		assert.Equal(t, "111", collector.Section("DEFAULT").Key("analytics_config_audit_ttl").String(), "Invalid analytics_config_audit_ttl")
+		assert.Equal(t, "444", collector.Section("DEFAULT").Key("analytics_statistics_ttl").String(), "Invalid analytics_statistics_ttl")
+		assert.Equal(t, "333", collector.Section("DEFAULT").Key("analytics_flow_ttl").String(), "Invalid analytics_flow_ttl")
 	})
 }
 
```

Comment:
I think you can also remove the strings below this line as they are not used anymore.